### ISSUE TITLE
Scrolling does not activate "on the fly" when the cell becomes longer than a screen height during live coding

### ISSUE THREAD

**Issue by prosoitos** [damianavila/RISE#558]
During live coding, if a cell becomes longer than a screen height, scrolling does not become available "on the fly". Switching to another slide and back activates the scrolling capability, but it breaks the workflow and smoothness of the talk/workshop a fair bit. Without doing this, new cells and their output drift off the screen and become invisible.

If there is a way to activate scrolling as soon as the slide gets past a certain height, that would be great.

A potential alternative solution, which would be much easier to implement, would be to make all slides scrollable (if the scrollable option is selected of course), even those which are small enough to fit entirely on screen. If all slides are scrollable by default, regardless of their size, then there is no need to find a way to implement the change on the fly.

**Comment by prosoitos** [comment]
Of course, a workaround to make the slide scrollable from the start is to add a giant markdown cell with lots of `<br>`. But it is a pretty ugly solution because, even though that cell is invisible, the content of the slide is awkwardly pushed to the top of the slide.

**Comment by damianavila** [comment]
Scrolling has been always hard on RISE.
I do not see scrolling on the fly like something easy to implement... but let me think about that a little bit more.
Making an optional all scrollable slides could be an interesting workaround (and maybe easier to implement).

**Comment by prosoitos** [comment]
Thanks @damianavila !

I think that it would indeed be a lot easier to implement. That option certainly would work very well in my case!

**Comment by prosoitos** [comment]
Some users may not like that option because it would create a scroll bar on all the slides. But that can be easily address: I personally don't like having a scroll bar on slides as I find it cluttering, so I made it transparent in my css (that requires `!important` to override the default css).
